web-ui/src/main/resources/catalog/locales/en-search.json

```diff
@@ -307,5 +308,8 @@
   "wpsDescribeProcessFailed": "DescribeProcess failed",
   "wpsExecute": "Execute",
   "wpsExecuteFailed": "Execute failed",
-  "wpsErrorCodeReturned": "The server returned status code"
+  "wpsErrorCodeReturned": "The serveur returned status code",
+  "mdvalid": "Metadata is valid",
+  "mdinvalid": "Metadata is invalid",
+  "mdnovalidation": "Metadata has no validation information."
```

Comment:
The icon with the tooltip is displayed either if publication of invalid metadata is allowed or not. Using "Invalid metadata can't be publish" can be confusing if the publication of invalid metadata is allowed. 

Will check to distinguish the tooltip depending on the mode. I guess that would be better.